auth.go

```diff
@@ -134,12 +141,7 @@ func (m *MenderAuthManager) MakeAuthRequest() (*AuthRequest, error) {
 		return nil, errors.Wrapf(err, "failed to obtain device public key")
 	}
 
-	tentok, err := m.store.ReadAll(authTenantTokenName)
-	if err != nil {
-		return nil, errors.Wrapf(err, "failed to read tenant token")
-	}
-
-	tentok = bytes.TrimSpace(tentok)
+	tentok := bytes.TrimSpace([]byte(m.tenantToken))
```

Comment:
There is also `strings.TrimSpace` method. This will avoid converting this to `[]byte` back and forth.